io/disk/htx_block_devices.py

```diff
@@ -46,7 +46,8 @@ def setUp(self):
         """
         Setup
         """
-        if 'ppc64' not in process.system_output('uname -a', shell=True):
+        if 'ppc64' not in process.system_output('uname -a',
```

Comment:
As you are already handling, `distro.detect().arch` can be used which also avoids decoding here.